sign/schnorr_test.go

```diff
@@ -27,19 +27,38 @@ func TestSchnorrSignature(t *testing.T) {
 	larger := append(s, []byte{0x01, 0x02}...)
 	assert.Error(t, VerifySchnorr(suite, kp.Public, msg, larger))
 
+	constantBytes := []byte{243, 45, 180, 140, 73, 23, 41, 212, 250, 87, 157, 243,
```

Comment:
Oh, I see. just move the `// wrong challenge` line before the `constantBytes`, then we'll understand it, too ;)